apps/passwd.c

```diff
@@ -706,12 +706,11 @@ static char *shacrypt(const char *passwd, const char *magic, const char *salt)
     cp = out_buf + strlen(out_buf);
     *cp++ = ascii_dollar[0];
 
-# define b64_from_24bit(B2, B1, B0, N)                                   \
+# define b64_from_24bit(B2, B1, B0, N)                                  \
     do {                                                                \
         unsigned int w = ((B2) << 16) | ((B1) << 8) | (B0);             \
         int i = (N);                                                    \
-        while (i-- > 0)                                                 \
-            {                                                           \
+        while (i-- > 0) {                                               \
                 *cp++ = cov_2char[w & 0x3f];                            \
                 w >>= 6;                                                \
             }                                                           \
```

Comment:
Shouldn't these lines be re-aligned too?

Reply:
Eventually, but so far I have been addressing specific classes of issues reported by `checkpatch.pl`. I'll run `checkpatch.pl` again once this is merged, to catch more issues.

Is it OK to rebase to fix conflicts?

Reply:
Yes, please rebase

Reply:
Done